.github/workflows/build_python_wheels.yml

```diff
@@ -7,8 +7,7 @@ on:
       - 'releases/**'
   pull_request:
     branches:
-      - master
-
+      - '*'
```

Comment:
Is it intended that you will build on PR to any branch? Or am I misunderstanding what is going on here?

Reply:
Yes that is intended. This is useful for example for a feature branch